help/assets/css/styles.scss

```diff
@@ -0,0 +1,273 @@
+---
+---
+@import url('./fonts.css');
+
+:root {
+    --color-dark: #0B1B34;
+    --spacing-default: 24px;
+}
+
+* {
+    margin: 0;
+    border: 0;
+    font: inherit;
+    font-size: 100%;
+    vertical-align: baseline;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+html {
+    line-height: 1;
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+caption,
+th,
+td {
+    text-align: left;
+    font-weight: 400;
+    vertical-align: middle;
+}
+
+q,
+blockquote {
+    quotes: none;
+}
+
+q:before,
```

Comment:
Oh, this would be another good time to make it a bit more sassy:

```scss
q,
blockquote {
    quotes: none;
    
    &:before,
    &:after {
        content: none;
    }
}
```

Reply:
Good catch updating it